waggle-dance-api/src/main/java/com/hotels/bdp/waggledance/api/model/MetastoreTunnel.java

```diff
@@ -26,12 +26,14 @@
 
   private static final int DEFAULT_PORT = 22;
   private static final String DEFAULT_LOCALHOST = "localhost";
+  private static final int DEFAULT_TIMEOUT_MILLIS = 60000; // 1 minute
 
   private @NotBlank @TunnelRoute String route;
   private @Min(1) @Max(65535) int port = DEFAULT_PORT;
   private String localhost = DEFAULT_LOCALHOST;
   private @NotBlank String privateKeys;
   private @NotBlank String knownHosts;
+  private @Min(0) int timeout = DEFAULT_TIMEOUT_MILLIS;
```

Comment:
I think we should either name this timeoutMillis to be clear what unit it is or we introduce two values here - one which is a TimeUnit and the other being the value.

Reply:
Even if the docs say the units is millis?

Reply:
Yeah, I guess it's fine as long as its clearly documented.